recipes/nsight-compute/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "nsight-compute" %}
+{% set version = "2022.4.0.15" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/nsight_compute/{{ platform }}/nsight_compute-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: f084e05eb4d2ba32aceb25e1dcfe03f2a50127630973722b65219cf9e986a139  # [linux64]
+  sha256: 20e58ce79681bc8fd39394cfb8f8316c177fe4175af3ae95c025996f45904732  # [ppc64le]
+  sha256: 7d0b3d4d01ce36657fa739496c7b0a9c627f5fa42021c1696ddd15e119bb05a4  # [aarch64]
+  sha256: 958da9986841c49cb5a2885d1e14e4c673ba94e4b404ef9389d083b7a0095d84  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  missing_dso_whitelist:
+    - "*"
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - sysroot_{{ target_platform }} 2.17
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+  host:
+     - cuda-version {{ cuda_version }}
+  run:
+     - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+  run_constrained:
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+
+test:
+  commands:
+    - test -f $PREFIX/bin/ncu                      # [linux]
+    - test -d $PREFIX/nsight-compute               # [linux]
+    - if not exist %PREFIX%\nsight-compute exit 1  # [win]
+
+about:
+  home: https://developer.nvidia.com/nsight-compute
+  license_file: LICENSE
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
+  license_url: https://docs.nvidia.com/cuda/eula/index.html
+  summary: NVIDIA Nsight Compute is an interactive kernel profiler for CUDA applications
+  description: |
+    NVIDIA Nsight Compute is an interactive kernel profiler for CUDA
+    applications on x86_64 platforms. It provides detailed performance
```

Comment:
"on x86_64 platforms": Does this mean that the Nsight Compute program can run on linux64/aarch64/ppc64le/win64 but it can only profile applications running on linux64? Or is this comment inaccurate, and we _can_ profile on aarch64/etc.?

Reply:
Probably best to leave these details up to their docs, which are better maintained. Their support matrix has the details: https://developer.nvidia.com/tools-overview/nsight-compute/get-started